pulsar-functions/worker/src/main/java/org/apache/pulsar/functions/worker/FunctionRuntimeManager.java

```diff
@@ -210,21 +210,25 @@ public FunctionRuntimeManager(WorkerConfig workerConfig, WorkerService workerSer
      * 2. After current assignments are read, assignments belonging to this worker will be processed
      */
     public void initialize() {
-        log.info("/** Initializing Runtime Manager **/");
         try {
-            this.functionAssignmentTailer = new FunctionAssignmentTailer(
-                    this,
-                    this.getWorkerService().getClient().newReader(),
-                    this.workerConfig,
-                    this.errorNotifier);
+
```

Comment:
I like that we are no longer using FunctionAssignmenttailer here. However maybe a static method that consolidates. this reader creation and the one in assignment tailer?

Reply:
sure